include/zephyr/net/wifi.h

```diff
@@ -322,10 +322,12 @@ enum wifi_ps {
 	WIFI_PS_ENABLED,
 };
 
+/** @cond INTERNAL_HIDDEN */
 static const char * const wifi_ps2str[] = {
 	[WIFI_PS_DISABLED] = "Power save disabled",
 	[WIFI_PS_ENABLED] = "Power save enabled",
 };
```

Comment:
actually... why do we have static arrays defined in headers? This should be in a C file...

Reply:
+1, I actually raised the issue to @krish2718 in this discussion https://github.com/zephyrproject-rtos/zephyr/pull/60686#discussion_r1289898763

Reply:
This originally was an `inline` function but was later changed to `static arrays`, I can move to `wifi_shell.c` and as extern (As they might be used by others).

Reply:
If it's is desirable it can be used by others, and for consistency, would it make sense to turn it into a `wifi_ps_txt()` helper function, similar to how it's done for e.g. https://github.com/zephyrproject-rtos/zephyr/blob/4cbb9d8f5f464325948b41ac24939d4434f6ab8c/include/zephyr/net/wifi.h#L52-L75

Thanks!

Reply:
`wifi_security_txt` this was the original code, and `wifi_ps_txt` was also added but due to a review comment went with static array.

Reply:
> So, will take care of converting this to static array and moving to C file.

Thanks! Please ping me when done so that I can update this PR accordingly.

Reply:
Sure, but lets not wait for that PR to be ready soon :), it might be a while.